tutorials/phasorpy_introduction.py

```diff
@@ -327,8 +328,49 @@
 # %%
 # Select phasor coordinates
 # -------------------------
+#
+# The :py:mod:`phasorpy.cursors` module provides functions for selecting phasor
+# coordinates to define and mask regions of interest within the phasor space.
+#
+# Mask regions of interest in the phasor space using circular cursors:
 
-# TODO
+from phasorpy.color import CATEGORICAL
+from phasorpy.cursors import mask_from_circular_cursor
+
+cursors_real = [0.69, 0.59]
+cursors_imag = [0.32, 0.33]
+radius = [0.05, 0.05]
+cursors_masks = mask_from_circular_cursor(
+    real, imag, cursors_real, cursors_imag, radius=radius
+)
+
+# %%
+# Plot the cursors in distinct colors:
+
+phasorplot = PhasorPlot(frequency=frequency, title='Cursors')
+phasorplot.hist2d(real, imag)
+for i in range(len(cursors_real)):
+    phasorplot.circle(
+        cursors_real[i],
+        cursors_imag[i],
+        radius=radius[i],
+        color=CATEGORICAL[i],
+        linestyle='-',
+    )
```

Comment:
At some point (not in this PR), we might want to overload the `PhasorPy.cursor` function to handle multiple cursors at once, or to introduce a new `PhasorPy.cursors` function.

Reply:
I think you meant `PhasorPlot.cursors`. If this is the case, then this is a good idea, it will simplify the tutorials and also some workflows since generally more than one cursor is selected in a single image.

Reply:
Yes, `PhasorPlot.cursors`. I will add this on my TODO list